server/controllers/admin/locations.js

```diff
@@ -499,3 +515,37 @@ exports.update.village = function updateVillage(req, res, next) {
     .catch(next)
     .done();
 };
+
+exports.delete = {};
+
+exports.delete.country = (req, res, next) => {
+  const sql = 'DELETE FROM country WHERE uuid=?';
+  const _uuid = db.bid(req.params.uuid);
+  db.exec(sql, _uuid).then(() => {
+    res.sendStatus(200);
```

Comment:
According to our [API Design Guidelines](https://github.com/IMA-WorldHealth/bhima/wiki/Design:-Server-CRUD-APIs#api-design), the DELETE request success status code should be 204.